src/patternfly/components/Compass/compass-footer.hbs

```diff
@@ -0,0 +1,6 @@
+<div class="{{pfv}}compass__footer{{#if compass-footer--modifier}} {{compass-footer--modifier}}{{/if}}"
```

Comment:
So I'm wondering whether some of these components should use their semantic elements (footer element here, and header and main elsewhere).

Issue is the compass__panel components living outside header, footer, and main. I'm not sure if putting them in `aside` elements would make the most sense (they sorta do I think).

Probably fine to leave them as plain divs for now, but worth mulling over some more, especially because I assume we need (or at least want) the structure of Compass to be as it is now: header, panel, main, panel, and footer all being adjacent siblings (rather than main containing the 2 panels).